cortx-motr.spec.in

```diff
@@ -127,7 +127,9 @@ BuildRequires:  systemd-devel
 %if %{rhel} < 8
 BuildRequires:  python-devel
 %else
+# @todo Deprecate python2
 BuildRequires:  python2-devel
+BuildRequires:  python3-devel
```

Comment:
Don't we need python2 for spiel anymore?

Reply:
Keeping python2-devel, until spiel is ported to python3, https://github.com/Seagate/cortx-motr/pull/1375